src/events.js

```diff
@@ -316,28 +299,34 @@ function fireEvent(element, event) {
   return element.dispatchEvent(event)
 }
 
-Object.entries(eventMap).forEach(([key, {EventType = Event, defaultInit}]) => {
```

Comment:
They were just defaulting to `Event` before due to `EventType = Event` in the function params above, when `EventType` was `undefined` (ie, `InputEvent` didn't exist before jsdom 12 so was `undefined`). Now that they are all changed to strings, like `'InputEvent'`, it's always defined so never defaulted over to `'Event'`.

Your new fix of
```
if (typeof window[EventType] === 'undefined') {
        window[EventType] = class extends window.Event {}
}
const event = new window[EventType](eventName, eventInit);
```
works, but we could also just do something like
```
const eventType = window[EventType] || window.Event;
const event = new eventType(eventName, eventInit);
```
if we wanted to keep the same behavior of not shimming.

Reply:
Yeah, my changes address this.

Reply:
oh, cool. Bit behind on the code updates then ^_^
I definitely like name `EventConstructor` over a weird lowercased `eventType`.